src/autorest-core/lib/pipeline/plugins/autorest-dotnet.ts

```diff
@@ -3,10 +3,129 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
handled by https://github.com/Azure/autorest/pull/2182/files#diff-10eaf6f76ff8877812109b5a5a608955R59 and calling the plugin on `config.GetPluginViews("azure-validator-individual")` right [here](https://github.com/Azure/autorest/pull/2182/files#diff-a6dae286c5e4ee8eac28f0dbb0822f1eR161)